comment.go

```diff
@@ -140,6 +141,35 @@ func (f *File) AddComment(sheet, cell, format string) error {
 	return err
 }
 
+// DelComment provides the method to delete comment in a sheet by given worksheet
+//
+//	err := f.DelComment("Sheet1", "A30")
+func (f *File) DelComment(sheet, cell string) (err error) {
+	sheetFile, sheetExist := f.sheetMap[sheet]
```

Comment:
Use the `workSheetReader` to check the worksheet if exists, and you will also get the pointer of the deserialized worksheet structure. This pull request won't remove the `legacyDrawing` of the worksheet when all comments on the worksheet have been removing. I think we need to remove it, so receive the pointer of the deserialized worksheet structure here to do that, and you needn't create a new error for not exist worksheet. Note that the exported errors should be defined in the `errors.go`.